edgePy/_DGEList.py

```diff
@@ -116,10 +116,20 @@ def samples(self) -> np.array:
 
     @samples.setter
     def samples(self, samples: Optional[np.ndarray]) -> None:
-        # TODO: Validate samples here
```

Comment:
While you're at it could you use the same logic and validate `self.genes` on 142?

https://github.com/r-bioinformatics/edgePy/pull/39/files#diff-91f57b81ee13bc2b01c571c9d899cd82R142

Reply:
Absolutely! I'll do that in a separate pull request so that this one stays separate, I guess